en/frames_rover/aion_r1.md

```diff
@@ -0,0 +1,78 @@
+# Aion Robotics R1 UGV
+
+<Badge type="warning" text="main (PX4 v1.15)" />
+
+This vehicle was chosen to test and improve the differential drive support for PX4.
+
+Currently, PX4 supports MANUAL mode for this setup.
+
+![Aion Robotics R1 UGV](../../assets/airframes/rover/aion_r1/r1_rover_no_bg.png)
+
+## Parts List
+
+- [Aion R1 (Discontinued)](https://www.aionrobotics.com/)
+  - [Documentation](https://github-docs.readthedocs.io/en/latest/r1-ugv.html)
+- [RoboClaw 2x15A](https://www.basicmicro.com/RoboClaw-2x15A-Motor-Controller_p_10.html)
+  - [R1 Roboclaw specifications](https://resources.basicmicro.com/aion-robotics-r1-autonomous-robot/)
+- [Auterion Skynode](../companion_computer/auterion_skynode.md)
+
+## Assembly
+
+The assembly consists of a 3D-printed frame on which all the autopilot parts were attached.
+For this build this includes an [Auterion Skynode](../companion_computer/auterion_skynode.md), connected to a Pixhawk Adapter Board that interfaces with the RoboClaw motor controllers over serial.
+
+![R1 Assembly](../../assets/airframes/rover/aion_r1/r1_assembly.png)
+
+:::note
+If using a standard Pixhawk you could connect the RoboClaw to the Autopilot without an Adapter Board.
+:::
+
+The RoboClaw should be connected to a suitable suitable serial (UART) port on the flight controller, such as `GPS2` or `TELEM1`.
+Other RoboClaw wiring is detailed in the [RoboClaw User Manual](https://downloads.basicmicro.com/docs/roboclaw_user_manual.pdf) 'Packet Serial Wiring' section and shown below (this setup has been validated for compatibility).
+
+![Serial Wiring Encoders](../../assets/airframes/rover/aion_r1/wiring_r1.jpg)
+
+## PX4 Configuration
+
+### Rover Configuration
+
+Use _QGroundControl_ for rover configuration:
+
+1. In the [Basic Configuration](../config/README.md) section, select the [Airframe](../config/airframe.md) tab.
+1. Choose **Aion Robotics R1 UGV** under the **Rover** category.
+
+![Select Airframe](../../assets/airframes/rover/aion_r1/r1_airframe.png)
+
+### RoboClaw Configuration
+
+First configure the serial connection:
+
+1. Navigate to the [Parameters](../advanced_config/parameters.md) section in QGroundControl.
+
+   - Set the [RBCLW_SER_CFG](../advanced_config/parameter_reference.md#RBCLW_SER_CFG) parameter to the serial port to which the RoboClaw is connected (such as `GPS2`).
+   - [RBCLW_COUNTS_REV](../advanced_config/parameter_reference.md#RBCLW_COUNTS_REV) specifies the number of encoder counts required for one wheel revolution.
+     This value should be left at `1200` for the tested `RoboClaw 2x15A Motor Controller`.
+     Adjust the value based on your specific encoder and wheel setup:
+   - If using several RoboClaw motor controllers, each can be assigned a unique address on the bus, with the default address being 128.
+     To set a different address, specify it in the [RBCLW_ADDRESS](../advanced_config/parameter_reference.md#RBCLW_ADDRESS) parameter.
```

Comment:
This confuses me. If I have several motor controllers, and each must have its own address, how do I set them when I only have one parameter - `RBCLW_ADDRESS`?

FYI only, note that I do lots of cross linking, because it makes it easy to spot when people change the underlying implementations for parameters etc (since these often disappear).

Reply:
@MaEtUgR This is actually a good question. Perfect that we ordered a second roboclaw, I'll test this next week.

Reply:
So this looks good to me once this issue is resolved.

Reply:
@hamishwillee You are asking exactly the right questions. To clarify: You need the correct Roboclaw address in the transaction header even if you only have one ESC. It needs to be the same that you configured with the Roboclaw configuration tool before over USB directly. We currently only have one ESC in our vehicle. The reason I would not spend too much time investigating yet is that AFAIK we are so far the only Roboclaw users on PX4. The driver improvements and docs should facilitate other users to get using [it](https://www.basicmicro.com/motor-controller) and the Aion R1/[R6](https://www.aionrobotics.com/r6).

Using multiple ESCs on one vehicle was never tested and we should make that clear. We won't invest time into doing that until we need the functionality e.g. to make a Mecanum wheel Rover where all wheels need a separate controller. We'd have the hardware parts for that but it's currently not the focus/priority.

Reply:
Thanks @MaEtUgR (and @PerFrivik ). So:
1. Then we don't support multiple RoboClaw MCs, because there is only one param for setting the header sent by PX4. I have made this clear. Also noted that you should not have to change this, but if you do, then you must make the param match.
2. I have added the information about Aion R6 to the intro too. I was wondering why we were using a discontinued frame. Helpful to be able to to day "this is still very useful".

I will merge, but if you have any issues with this change, please add a post PR to fix.

Reply:
Thank you @hamishwillee ! I appreciate the help on getting this in.

Reply:
No worries. It's pretty cool! Rover is much under loved.